server/chat-plugins/scavengers.js

```diff
@@ -33,6 +34,11 @@ const HISTORY_PERIOD = 6; // months
 
 const ScavengerGames = require("./scavenger-games");
 
+let scavengersData = {};
+try {
+	scavengersData = require(`../../${DATABASE_FILE}`);
```

Comment:
Please use `FS` instead, `require` caching will lead to confusion when hotpatching.